impl/src/main/java/io/kroxylicious/testing/kafka/invm/InVMKafkaCluster.java

```diff
@@ -427,4 +429,16 @@ public synchronized EndpointPair getEndpointPair(Listener listener, int nodeId)
         var port = portsAllocator.getPort(listener, nodeId);
         return EndpointPair.builder().bind(new Endpoint("0.0.0.0", port)).connect(new Endpoint("localhost", port)).build();
     }
+
+    private static void trapKafkaSystemExit() {
+        Exit.setExitProcedure((statusCode, message) -> {
+            final IllegalStateException illegalStateException = new IllegalStateException(message);
+            System.out.println("Kafka tried to exit with statusCode: " + statusCode + " and message: " + message);
```

Comment:
System.out is not a long term solution but until we can confirm that the logger works its staying here.